tests/BasicSearchTests.FunctionalTests.Core/V3SearchFunctionalTests.cs

```diff
@@ -37,5 +37,23 @@ public class V3SearchFunctionalTests : BaseFunctionalTests
             Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task ResultForJsonNetHasDownloads()
+        {
+            // Act
+            var response = await Client.GetAsync(new V3SearchBuilder() { Query = "JSON.Net" }.RequestUri);
```

Comment:
I agree that this should be a blank query that verifies the packages in the response don't have 0 download counts.

Not sure if all of our gallery functional tests will always succeed when we run them locally anyway...it's fine if this fails but ideally if we could write it in a way that it wouldn't that would be great.